racket/collects/racket/contract/private/provide.rkt

```diff
@@ -983,16 +1026,6 @@
                                               field-contract-ids ...))
                        (provide (rename-out [-struct:struct-name struct:struct-name]))))))))))
 
-       (define (map/count f . ls)
-         (let loop ([ls ls]
-                    [i 0])
-           (cond
-             [(andmap null? ls) '()]
-             [(ormap null? ls) (error 'map/count "mismatched lists")]
```

Comment:
Removing `map/count` removes this error check.

@rfindler let me know if I should revert & keep `map/count`